### DISCUSSION TITLE
Unable to update dependency via yarn up on berry version

### DISCUSSION THREAD

**Discussion by helltraitor** [yarnpkg/berry#5538] [Q&A]
I have `^3.6.0` version requirement but `yarn up` doesn't install a new one when it's available `3.6.1`
As for me, it seems like broken feature especially when `--help` says, that `yarn up` === `yarn upgrade --latest`

So my question is... How to update?
I don't want to use interactive tools since they change `package.json` file (I would even like to use `^x.x.0` instead of changing `package.json` file every time)

Removing lock file is solving issue, but it's not what I wanted (it's crutch)

**Comment by arcanis** [comment]
> I have ^3.6.0 version requirement but yarn up doesn't install a new one when it's available 3.6.1

You didn't put a repro, so we can't help. `yarn up` already does what you suggested.

**Reply by helltraitor** [reply]
As for mine machine (and sandbox), you can clone my [repo](https://github.com/helltraitor/helltraitor.github.io), checkout on `HEAD~1` (before manual update). Run `yarn install && yarn info nuxt version`, then `yarn up && yarn info nuxt version`. You will see same `nuxt 3.6.0`

I'm not sure how I can __reproduce__ this behavior, but I've tried on my linux WSL and it's same
![image](https://github.com/yarnpkg/berry/assets/95148327/58b4dc32-8688-4c3e-a9f6-cd4bb3b9ff28)
